library/core/src/main/java/com/google/android/exoplayer2/extractor/flv/VideoTagPayloadReader.java

```diff
@@ -80,6 +80,9 @@ protected boolean parseHeader(ParsableByteArray data) throws UnsupportedFormatEx
   protected void parsePayload(ParsableByteArray data, long timeUs) throws ParserException {
     int packetType = data.readUnsignedByte();
     int compositionTimeMs = data.readUnsignedInt24();
+    // compositionTimeMs is signed int 24, change unsigned int 24 to signed int 24
```

Comment:
Can you add a readInt24 method to ParsableByteArray instead (preferably with unit tests in ParsableByteArrayTest)? It seems like an omission; that class has methods for reading both signed and unsigned values of other lengths.

Reply:
I have changed as you said. Please review.